docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+
+For gates tests, the goal is to ensure gates are properly implemented. Sometimes gate operations in certain simulators are in reverse order (compared to Orquestra Core and other frameworks). When you use a gate operation in Orquestra, it's important to make sure it's represented correctly in the simulator and/or backend. In order to do that, Orquestra Core has a `suite of gate operations <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/testing/test_cases_for_backend_tests.py>`_ that should be tested on each backend and simulator. This is done with `QuantumBackendGateTests <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/api/backend_test.py#L193=>`_ and `QuantumSimulatorGateTests <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/api/backend_test.py#L432=>`_
+
+.. TODO: updating the link to the backend test excluding RH gates to braket when public
+
+There is also the case where a certain backend doesn't use some gates that we support. When this happens, in the backend test, a list of excluded gates will be used in the ``QuantumSimulatorGatesTest`` like in `TestCirqSimulatorGates <https://github.com/zapatacomputing/orquestra-cirq/blob/main/tests/orquestra/integrations/cirq/simulator/simulator_test.py#L147>`_. For example, that might look something like this:
+
+.. code-block:: python
+
+  class TestCirqSimulatorGates(QuantumSimulatorGatesTest):
+      gates_to_exclude = ["RH"]
+      pass
+
+.. _backend_methods:
+
+Backend Methods Usage
+=====================
+
+In order to use quantum hardware, the user must use the backend's authentication method to prove that they have the right credentials to send instruction to the hardware. The authentication method is specified by the hardware provider. Providing authentication credentials is the first step in using ``QuantumBackend``. Below is an example of using IBM's hardware through ``QiskitBackend``.
+
+``QuantumSimulator``\ s run in your own environment and therefore can be used without needing authentication credentials.
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend creation example
+  :end-before: # End QuantumBackend creation example
+
+General Backend Methods
+-----------------------
+
+After intializing the backend we can send the ``Circuit`` and number of repetitions to the backend and read the measurement. For more information on creating circuits to send to a backend, refer to the :ref:`Circuits Guide <circuits_guide>`. For a single circuit, that is executed as follows:
```

Comment:
Suggestion?
```suggestion
After intializing the backend we can send the ``Circuit`` and number of repetitions to the backend and read the measurement. For more information on creating circuits to send to a backend, refer to the :ref:`Circuits Guide <circuits_guide>`. For a single circuit, it looks as follows:
```

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64 with slightly different language :)